### PR TITLE
[issue#11] Fix navbar / add register

### PR DESCRIPTION
**Pull request by nikitaermishin**
Added register to navbar, created register page, added register page tests

### CODE REVIEW

**Comment by atb00ker** [comment]
I am not a mentor for this task and I don't know much about the changes but we need to ensure clean commit history so that we understand our changes while releasing and / or tracking a bug in the future, so please help us keeping the commit history clean and submit only one commit that follows the guidelines.

Commit guidelines here: http://openwisp.io/docs/developer/contributing.html#how-to-commit-your-changes-properly

**Comment by atb00ker** [comment]
P.S: Code would be reviewed by @NoumbissiValere

File: src/components/Register/RegisterForm.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+const RegisterForm = props => {
+  return (
+    <div>
+      <div className="container">
+        <h1>Register</h1>
+        <hr />
+        <Card style={{padding:'10px 20px', width: '35rem'}} className="mx-auto">
+          <Tabs style={{padding:'0 10px'}} className="mx-auto" defaulActiveKey="login">
+            <Tab eventKey="login" title="Login">
+              <hr />
```
Comment (NoumbissiValere) [inline src/components/Register/RegisterForm.js]: Please can you correct the code indentation? this will improve code readability and ease debugging

**Review by NoumbissiValere — CHANGES_REQUESTED** [review]

File: src/components/Register/RegisterForm.js (hunk 1 above)
Comment (nikitaermishin) [reply src/components/Register/RegisterForm.js]: Ok, will correct it

**Comment by NoumbissiValere** [comment]
Great job @nikitaermishin  Please you can move on to another task while i confirm that everything still works fine

**Review by NoumbissiValere — APPROVED** [review]

**Pull request closed without merging**